### DISCUSSION TITLE
Cannot create Case - Response 400

### DISCUSSION THREAD

**Discussion by a3hiennt** [Security-Onion-Solutions/securityonion#8228] [Unsupported Versions]
Hi Team.
I have issue related to Case.
When I create new Case or perform Escalate. I receive status code 400
![image](https://user-images.githubusercontent.com/80515844/176836038-09f1700d-a1fd-454d-91cb-786ec59cb61e.png)
I have checked log file and received bellow message.
`timestamp=2022-07-01T06:23:00.400672367Z level=warn message="User not found in context"
timestamp=2022-07-01T06:23:00.412829508Z level=warn message="User not found in context"
timestamp=2022-07-01T06:23:00.64922132Z level=warn message="Request did not complete successfully" error="Object not found" requestId=bcf774e3-34ac-4f44-8943-1474974e03a4 requestor=bfd2295a-8b03-4df5-9ca0-f80d5f9d575f`
Please help me resolve this issue.
Thanks so much.

**Comment by dougburks** [comment]
From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

> Please include the following details where you can:
Security Onion version as seen in the lower left corner of SOC and in /etc/soversion. For example: 2.3.110
Is this a cloud deployment or on-prem? If on-prem, do you have Internet access or this an airgap installation?
Did you install from our Security Onion ISO image or did you perform a network installation?
If network installation, did you install on CentOS 7 or Ubuntu?
How many nodes do you have?
What are the hardware specs of each of those nodes?
How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)

Also, have you made any custom changes to the system?
